js/bytetrade.js

```diff
@@ -874,15 +875,15 @@ module.exports = class bytetrade extends Exchange {
         const amountTruncated = this.amountToPrecision (symbol, amount);
         const amountTruncatedPrecise = new Precise (amountTruncated);
         amountTruncatedPrecise.reduce ();
-        amountTruncatedPrecise.decimals -= baseCurrency['precision'];
+        amountTruncatedPrecise.decimals -= this.precisionFromString (baseCurrency['precision']);
```

Comment:
We can't use precisionFromString here, cause precision is a number, not a string. It has to be converted to a string first, or we should apply some other method here.

Reply:
I've fixed that atm with doing `numberToString` at first. is that correct fix for this case? I ask because will all cases of `0.0000xyz` ticksize converted to the correct string with same representation (without being ended up like: `0.0001000000000043534563` )?